StudioAspen/AAAGame2023-2024
# Mouse Sensitivity Slider Prefab

- added `SetMKSensitivity(float xSensitivity, float ySensitivity);` to PlayerInput

- added a SensitivityAdjuster script with an `AdjustOverallSensitivity(float sensitivityMultiplier)` function that will change the sensitivity based on a given float value. it changes it based on a set initial sensitivity for both the X and Y values, so the given float value will always produce the same sensitivity no matter what the current sensitivity is.

- Connected this to a UI slider prefab, slide it up to increase sensitivity, slide it down to decrease. the min and max is determined by the slider.

To Test:
- go to any scene and simply put the SensitivityUI Prefab into the canvas. It should work without issue. You'll have to slide the slider in the inspector since the mouse currently disappears when clicking in the game, but this obviously wont be an issue once a proper pause menu is implemented

NOTE:
- I dont believe the player's set slider value will persist across scenes, since it resets to the initial values on start(). that will need to be implemented
